Documentation/Functions/Typolink.rst

```diff
@@ -543,27 +543,30 @@ userFunc
     :name: typolink-userFunc
     :type: :ref:`data-type-function-name`
 
-    This passes the link-data compiled by the typolink function to a user-
-    defined function for final manipulation.
+    This passes the link-data compiled by the :typoscript:`typolink` function
+    to a user-defined PHP function for final manipulation.
 
     The :php:`$content` variable passed to the user-function (first parameter) is
```

Comment:
The first parameter `$content` to the userfunc is an object of type `LinkResultInterface`, it contains all pre-processed data of the URI you know of `typolink` property like page resolvement, target, class, section and mail handling.

Reply:
The second parameter `$conf` contains the original TypoScript configuration of the `typolink` property.